src/renderer/views/Playlist/Playlist.scss

```
@@ -30,6 +33,30 @@
   align-items: center;
 }
 
+.playlistItem-move ,
+.playlistItem-enter-active,
+.playlistItem-leave-active {
+  transition: all 0.2s ease;
+
+  // Hide action buttons during transitions
+  //
+  // The class for icon container is mainly styled in `_ft-list-item.scss`
+  // But the transition related classes are all on container elements
+  // So `:deep` is used
+  :deep(.ft-list-item .videoThumbnail .playlistIcons) {
+    display: none;
+  }
+  // Prevent link click
+  :deep(.ft-list-item .videoThumbnail .thumbnailLink) {
+    // https://developer.mozilla.org/en-US/docs/Web/CSS/pointer-events
+    pointer-events: none;
+  }
+}
+.playlistItem-enter, .playlistItem-leave-to {
+  opacity: 0;
+  transform: translateX(10%);
```

```suggestion
  translateX(calc(10% * var(--horizontal-directionality-coefficient))
```

(disclaimer: validate in local before trying; I sometimes make mistakes typing in the GH UI)

---

Seems missing `transform: ` prefix, and semicolon at the end...
Anyway I updated and validated it locally but you should check again

---

Thank you lol. Tested by adding/removing in English and Arabic. Looks good now!